dev/fattest.simplicity/src/componenttest/annotation/ExpectedFFDCs.java

```diff
@@ -0,0 +1,23 @@
+/*******************************************************************************
+ * Copyright (c) 2020 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package componenttest.annotation;
+
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+@Target({ ElementType.METHOD })
+@Retention(RUNTIME)
+public @interface ExpectedFFDCs {
```

Comment:
instead of making a public `@ExpectedFFDCs` annotation, lets use a package-protected annotation as an inner class as described in this article:
https://dzone.com/articles/repeatable-annotations-in-java-8-1
That way users won't accidentally import/use `@ExpectedFFDCs`

Reply:
This will also simplify the annotation processing code because you can just use `getAnnotationsByType` everywhere and not have to explicitly check for `@ExpectedFFDC` and `@ExpectedFFDCs` separately